### PR TITLE
universal react-app example

### LINKED ISSUE #23: Universal react app example
- redux connection

### PR DESCRIPTION
**Pull request by awinogradov**
It's example of universal react-app wich based on [babel-plugin-bem-import](https://github.com/awinogradov/babel-plugin-bem-import).

Close #23

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,4 +1,31 @@
...
+            "./tests/desktop.blocks"
+          ],
+          "techs": ["js", "css"]
+        }]
+      ]
+    },
+    "server": {
+      "plugins": [
+        ["bem-import", {
+          "levels": [
```
Comment (veged) [inline .babelrc]: let's do `.babelrc.js` for avoid copy-paste and for future usage for examples

Comment (awinogradov) [inline .babelrc]: It's imposiible now, sorry ;( Babel doesn't allow CommonJS configs ;(

**Comment by vkhv** [comment]
Why isn't used koajs? Is it difficult to integrate?)

**Comment by awinogradov** [comment]
Because it's simple example of server-side rendering. You can use it with any frameworks.

**Comment by awinogradov** [comment]
Everybody can understand this thing from specs on Jest in root of repo

**Pull request closed without merging**